packages/requests/src/lib/requests-result.ts

```diff
@@ -172,21 +172,30 @@ interface Options {
   staleTime?: number;
   // Ignore everything and perform the request
   skipCache?: boolean;
+  // Skip the request if it's already loading
+  skipWhileLoading?: boolean;
 }
 
 export function trackRequestResult<TData>(
   key: unknown[],
   options?: Options
 ): MonoTypeOperatorFunction<TData> {
   return function (source: Observable<TData>) {
-    return getRequestResult(key).pipe(
+    return getRequestResult(key, {
+      initialStatus: options?.skipWhileLoading ? 'idle' : undefined,
```

Comment:
It has to be idle for the first call to trigger the request. If the state is initially loading the request will be skipped.

Another option to solve this would be to extend the status info with a initial flag, or a pending requests count.

Reply:
Why not check if the key is in the isLoading state?

Reply:
Since isLoading is initialized with true by default any request would be skipped

Reply:
That's correct. We can add a fetchStatus with the values fetching or idle and use this.